install/pre-installation.md

```diff
@@ -522,38 +522,58 @@ Run the following steps before starting any of the system configuration procedur
 
 1. (`pit#`) Download the SHCD to the `prep` directory.
 
-   This will need to be retrieved from the administrators Cray deliverable.
+    This will need to be retrieved from the administrators Cray deliverable.
 
 1. Validate the SHCD.
 
-   See [Validate SHCD](../operations/network/management_network/validate_shcd.md) and then return to this page.
+    See [Validate SHCD](../operations/network/management_network/validate_shcd.md) and then return to this page.
 
 ### 3.2 Generate topology files
 
-1. (`pit#`) Generate `hmn_connections.json`.
+The following steps use the new, automated method for generating files. The previous step for
+[validate SHCD](#31-validate-shcd) will have generated "paddle" files, these are necessary for generating
+the rest of the seed files.
```

Comment:
Is this still true? The steps below are the old manual way.

Reply:
Needs rewording, the notice I added contradicts this.